content/02.introduction.md

```diff
@@ -1,6 +1,19 @@
-#Introduction
-Set introduction for paper here
-Talk about problem, goal, and significance of paper
+# Introduction
+
+Populating knowledge bases is a daunting and arduous task that takes a significant amount of manual effort to accomplish.
+Typically, this task involves  manual curation, which is the process of curators reading and parsing published papers to extract relationships of interest. 
+This framework becomes infeasible as the rate of publications continues to increase exponentially [@doi:10.1002/asi.23329].
```

Comment:
```suggestion
The rate of publications is increasing exponentially [@doi:10.1002/asi.23329], but the number of curators is not.
We expect this will lead to gaps in knowledge bases.
```